packages/manager/modules/pci/src/projects/projects.constant.js

```diff
@@ -1,7 +1,26 @@
 export const DEFAULT_PROJECT_KEY = 'PUBLIC_CLOUD_DEFAULT_PROJECT';
 export const QUOTA_THRESHOLD = 80;
 
+export const ORDER_FOLLOW_UP_POLLING_INTERVAL = 7000;
+
+export const ORDER_FOLLOW_UP_STATUS_ENUM = {
+  DOING: 'DOING',
+  DONE: 'DONE',
+  ERROR: 'ERROR',
+  TODO: 'TODO',
+};
+
+export const ORDER_FOLLOW_UP_STEP_ENUM = {
+  AVAILABLE: 'AVAILABLE',
+  DELIVERING: 'DELIVERING',
+  VALIDATED: 'VALIDATED',
```

Comment:
Still needed even if you fetch them from API ?

Reply:
No, i made some refactor part.
Because there are others part where these const are imported.
I made a refactor commit for that.